sycl/test/basic_tests/version.cpp

```diff
@@ -0,0 +1,22 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %RUN_ON_HOST %t.out
+
+#include <CL/sycl.hpp>
+#include <iostream>
+
+int main() {
+#ifndef __LIBSYCL_MAJOR_VERSION
+  std::cerr << "__LIBSYCL_MAJOR_VERSION was not found\n";
```

Comment:
Can the `cerr`'s be replcaed with a mere `#error`. No `RUN` line will be needed then.